### PR TITLE
feat: snapshot library CH column migration

### PR DESCRIPTION
**Pull request by pauldambra**
Splitting just the clickhouse migration out of https://github.com/PostHog/posthog/pull/26747

This adds a new column to session_replay_events that will help us track usage as we add mobile replay billing

The SDKS send lib_version as well, but in the short to medium term we don't _need_ this on the table, so I'm not going to add it now

I've tested this by 

* applying just the migration and **not** sending $lib and seeing we can ingest events
* applying just the migration (in the parent PR) and sending $lib and seeing we can ingest events
* migrating from scratch and **not** sending $lib and seeing we can ingest events
* migrating from scratch (in the parent PR) and sending $lib and seeing we can ingest events

### CODE REVIEW

File: posthog/api/test/__snapshots__/test_decide.ambr
Comment (pauldambra) [inline posthog/api/test/__snapshots__/test_decide.ambr:1]: 🤷 nothing to do with this PR 🤷

Comment (fuziontech) [reply posthog/api/test/__snapshots__/test_decide.ambr:1]: image

File: posthog/clickhouse/migrations/0095_add_snapshot_library_tracking.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+operations = [
+    # we have to drop materialized view first so that we're no longer pulling from kakfa
+    # then we drop the kafka table
+    run_sql_with_exceptions(DROP_SESSION_REPLAY_EVENTS_TABLE_MV_SQL()),
+    run_sql_with_exceptions(DROP_KAFKA_SESSION_REPLAY_EVENTS_TABLE_SQL()),
+    # now we can alter the target tables
+    run_sql_with_exceptions(ADD_LIBRARY_DISTRIBUTED_SESSION_REPLAY_EVENTS_TABLE_SQL()),
+    run_sql_with_exceptions(ADD_LIBRARY_WRITABLE_SESSION_REPLAY_EVENTS_TABLE_SQL()),
+    run_sql_with_exceptions(ADD_LIBRARY_SESSION_REPLAY_EVENTS_TABLE_SQL()),
```
Comment (fuziontech) [inline posthog/clickhouse/migrations/0095_add_snapshot_library_tracking.py:22]: I would add the column first to the `sharded_` table here, then the distributed tables. 
```suggestion
    run_sql_with_exceptions(ADD_LIBRARY_SESSION_REPLAY_EVENTS_TABLE_SQL()),
    run_sql_with_exceptions(ADD_LIBRARY_DISTRIBUTED_SESSION_REPLAY_EVENTS_TABLE_SQL()),
    run_sql_with_exceptions(ADD_LIBRARY_WRITABLE_SESSION_REPLAY_EVENTS_TABLE_SQL()),
```

**Comment by fuziontech** [comment]
Looks good, just that one comment. I've added the columns to the cluster with no issues! 

🚢 IT!

**Review by fuziontech — APPROVED** [review]

**Pull request merged**